vanessaspear/passport-client
# Updated auth components and Navbar formatting

## Description
Updated the auth components to use the passport_token sent from the server for authentication.  Installed bootstrap and added the cdn link/script to the index.html file.

Related Issues
- Closes #1

Testing
```
git fetch origin vs-navbar-formatting
git checkout vs-navbar-formatting
```
- Change the viewport size to small and verify the hamburger icon displays in the navbar
- Change the viewport to large and verify the hamburger icon disappears 
- Click login and try to login as a user.  Verify login works. 
- Click register and register as a new user.  Verify registrations works.
